airflow/utils/operator_helpers.py

```diff
@@ -164,8 +164,10 @@ def determine(
 
     def unpacking(self) -> Mapping[str, Any]:
         """Dump the kwargs mapping to unpack with ``**`` in a function call."""
-        if self._wildcard and isinstance(self._kwargs, Context):
-            return lazy_mapping_from_context(self._kwargs)
+        # We have a typestub that says Context is a TypedDict, but at runtime it's actually a custom
+        # MutableMapping class.
+        if self._wildcard and isinstance(self._kwargs, Context):  # type: ignore
+            return lazy_mapping_from_context(self._kwargs)  # type: ignore
```

Comment:
What’s the error here? I was able to remove both `type: ignore` comments locally and have pre-commit pass.

Reply:
I got an error saying something to the effect of "isistance doesn't work on TypedDict".